packages/esbuild-plugin/README.md

```diff
@@ -50,24 +50,105 @@ require("esbuild").build({
+
+#### <a name="include"></a>options.include:
+
+| Option             | Type                | Required | Description                                    |
+| ------------------ | ------------------- | -------- | ---------------------------------------------- |
+| paths              | `string[]`          | required | One or more paths to scan for files to upload. |
+| ignoreFile         | `string`            | optional | See above.                                     |
+| ignore             | `string`/`string[]` | optional | See above.                                     |
+| ext                | `array`             | optional | See above.                                     |
+| urlPrefix          | `string`            | optional | See above.                                     |
+| urlSuffix          | `string`            | optional | See above.                                     |
+| stripPrefix        | `array`             | optional | See above.                                     |
+| stripCommonPrefix  | `boolean`           | optional | See above.                                     |
+| sourceMapReference | `boolean`           | optional | See above.                                     |
+| rewrite            | `boolean`           | optional | See above.                                     |
+
+Example:
+
+```ts
+// esbuild.config.js
+const sentryEsbuildPlugin = require("@sentry/esbuild-plugin");
+
+require("esbuild").build({
+  plugins: [
+    sentryEsbuildPlugin({
+      // ...
+      include: [
+        {
+          paths: ["./packages"],
+          urlPrefix: "~/path/to/packages",
+        },
+        {
+          paths: ["./client"],
+          urlPrefix: "~/path/to/client",
+        },
+      ],
+    }),
+  ],
+});
+```
+
+#### <a name="setCommits"></a>options.setCommits:
+
+| Option         | Type      | Required  | Description                                                                                                                                                                                                                                                                                                                |
+| -------------- | --------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| repo           | `string`  | see notes | The full git repo name as defined in Sentry. Required if `auto` option is not `true`, otherwise optional.                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| repo           | `string`  | see notes | The full git repo name as defined in Sentry. Required if the `auto` option is not `true`, otherwise optional.                                                                                                                                                                                                                  |
```